serde_arrow/benches/groups/complex_common.rs

```diff
@@ -10,7 +10,7 @@ use serde::{Deserialize, Serialize};
 use serde_arrow::_impl::arrow2;
 
 #[derive(Debug, Serialize, Deserialize, ArrowField, ArrowSerialize, ArrowDeserialize)]
-struct Item {
+pub(crate) struct Item {
```

Comment:
I guess benchmarks cannot be used anyways, right? Maybe a simple pub is a bit shorter.

Reply:
done in local branch